Tasks/DockerV2/dockerbuild.ts

```diff
@@ -71,5 +68,51 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
     return dockerCommandUtils.build(connection, dockerFile, commandArguments, labelArguments, tagArguments, (data) => output += data).then(() => {
         let taskOutputPath = utils.writeTaskOutput("build", output);
         outputUpdate(taskOutputPath);
+
+        const builtImageId = getBuiltImageIdFromDockerBuiltOutput(output);
+        if (builtImageId && builtImageId != ""){
+            shareBuiltImageId(builtImageId);
+        }
     });
 }
+
+function shareBuiltImageId(builtImageId: string) {
+    let builtImages: string = tl.getVariable("DOCKER_TASK_BUILT_IMAGES");
+    const IMAGE_SEPARATOR_CHAR:string = ";";
+
+    if (builtImages && builtImages != ""){
+        builtImages += `${IMAGE_SEPARATOR_CHAR}${builtImages}`;
+    }
+    else {
+        builtImages = builtImageId;
+    }
+
+    tl.setVariable("DOCKER_TASK_BUILT_IMAGES", builtImages);
```

Comment:
I would like to learn how I can write a l0 that verifies that the env variable is getting the right value.

Reply:
The [documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#setvariable-initialize-or-modify-the-value-of-a-variable) says the variable is available as env variable to the tasks that follow.
Maybe test that the command to set the variable is present in `stdout`?

Also, can you pls test it out by actually running in a pipeline and adding a script to output this to the logs?